python/lsst/ip/diffim/dcrModel.py

```diff
@@ -558,50 +562,46 @@ def applyImageThresholds(self, image, highThreshold=None, lowThreshold=None, reg
             image[lowPixels] = lowThreshold[lowPixels]
 
 
-def applyDcr(maskedImage, dcr, warpCtrl, bbox=None, useInverse=False, splitSubfilters=False):
+def applyDcr(image, dcr, useInverse=False, splitSubfilters=False, **kwargs):
     """Shift a masked image.
 
     Parameters
     ----------
-    maskedImage : `lsst.afw.image.MaskedImage`
-        The input masked image to shift.
-    dcr : `lsst.afw.geom.Extent2I`
+    image : `numpy.ndarray`
+        The input image to shift.
+    dcr : `tuple` of two `float`
         Shift calculated with ``calculateDcr``.
-    warpCtrl : `lsst.afw.math.WarpingControl`
-        Configuration settings for warping an image
-    bbox : `lsst.afw.geom.Box2I`, optional
-        Sub-region of the masked image to shift.
-        Shifts the entire image if None (Default).
+        Uses numpy axes ordering (Y, X).
     useInverse : `bool`, optional
         Use the reverse of ``dcr`` for the shift. Default: False
```

Comment:
Is this really an "inverse", or a "reverse" or some other name? It's shifting by `-dcr`, no?

Reply:
Since it's a linear shift, a shift by `dcr` in one direction followed by a shift by `-dcr` along the same direction should recover the original image. So, I think it is appropriate to call it an inverse.

Reply:
The docstring says "reverse"?

Reply:
OK, I'll edit the docstring to make that consistent.